manifest.json

```diff
@@ -74,6 +74,23 @@
       "packageType": "Zip",
       "useCaseName": "Hello World Example"
     },
+    {
```

Comment:
This file should not be modified. For new templates, please modify `manifest_v2.json` instead.

Reply:
Moved template info to new manifest_v2.json file